help/en/releasenotes/current-draft-note.shtml

```diff
@@ -430,7 +430,11 @@
    <h3>Dispatcher</h3>
         <a id="Dispatcher" name="Dispatcher"></a>
         <ul>
-             <li></li>
+             <li>Display minimum scale speed  when activating train
+				Add basic column sorting on the traininfo selection screen, and correct missplaced columns.
+				Tidy on debug/trace in RosterSpeedProfile.
+				Make info message in allocated section to debug.
+			 </li>
```

Comment:
Since this is html, it will be a single line in the release note. If you want it on different lines, you can do this:
```
        <ul>
            <li>
                <ul>
                    <li>Display minimum scale speed  when activating train</li>
                    <li>Add basic column sorting on the traininfo selection screen, and correct missplaced columns.</li>
                    <li>Tidy on debug/trace in RosterSpeedProfile.</li>
                    <li>Make info message in allocated section to debug.</li>
                </ul>
            </li>
        </ul>
```